scripts/SetupServiceFabric/SetupServiceFabric.sh

```diff
@@ -3,8 +3,14 @@
 #
 # This script installs and sets up the Service Fabric Runtime and Common SDK.
 # It also sets up Azure Service Fabric CLI
-#
+
 # Usage: sudo ./SetupServiceFabric.sh
+# Above fetches Service Fabric Runtime and SDK along with required packages/dependencies from repositories and does the setup.
+# This script also supports installation of SDK from local .deb packages of Servie Fabric runtime and sdk. Both packages will be needed for installation and paths should be provided as parameters.
+# Below is the sample:
+# sudo ./SetupServiceFabric.sh --servicefabricruntime=/mnt/c/Users/sindoria/Downloads/servicefabric_8.2.142.2.deb --servicefabricsdk=/mnt/c/Users/sindoria/Downloads/servicefabric_sdkcommon_1.4.2.deb
+# In above scenario sf runtime is located at C:\Users\sindoria\Downloads\servicefabric_8.2.142.2.deb in windows host but in VM it will look like /mnt/c/Users/sindoria/Downloads/servicefabric_8.2.142.2.deb
```

Comment:
remove all this, and just say the files should be present at that path.

Reply:
updated the comment.